yabt/target_utils.py

```diff
@@ -174,6 +177,7 @@ def __init__(self, builder_name):
                 'fail_count': 0,
             },
             is_dirty=False,
+            tested={},
             _hash=None,
             _json=None)
```

Comment:
also need to add here `_test_json=None` (otherwise the `test_json` method might access an undefined attribute)

Reply:
Done.